### DISCUSSION TITLE
scrum prep Milestone 2 - 3

### DISCUSSION THREAD

**Discussion by Yahid1** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#126] [General]
Write your scrums here

**Comment by MrL9000** [comment]
I've finished most if not all of the menu view and just have to get in contact with the managers to go over the task, decide on a score, and close it.

**Comment by Janiel777** [comment]
No hice mucho esta semana. Solo trate de buscar un LTT pero aun no decido de que hacerlo.

**Comment by dAninAd1248** [comment]
This week I identified possible cases of second order ignorance, brainstormed possible LTTs, gave a blank stare at the code trying to understand it, and learnt a bit of angular.

**Comment by Yahid1** [comment]
This week I assigned some tasks as well as clarified some of the task objectives. Meet with team members to discuss the Software Architecture of the Dashboard and help them determine where they can work on tasks.

**Comment by AlexisTraversoPR** [comment]
Wasn't able to work much on the project, started brainstorming on possible lecture topic tasks

**Comment by Brandell13** [comment]
I completed both of my tasks and attended a meeting to ensure they met the necessary requirements.

**Comment by JaseyAcevedo** [comment]
Been searching for LTT and researching some companies for the job fair

**Comment by sebasagir** [comment]
this week I didn't do much related to the project other than look for lecture topic tasks

**Comment by rafa-quiles** [comment]
This week I attended meetings and worked on my assigned task, also looking into possible lecture topic tasks.

**Comment by Alej24** [comment]
Attended a team meeting and am deciding what LTT to do.  Additionally, I have an assigned task but I have to wait for something to be corrected before working on it.

**Comment by juan-ortiz92** [comment]
This week I worked with the implementation of announcements/alerts of notifications for the back end and worked on my second lecture topic task.

**Comment by marcosarraiza** [comment]
This week I had kind of a hell week, I just attended our weekly reunions and set up a meeting with a teammate to collaborate on a task.

**Comment by JoelTorres20** [comment]
This week I didn't do much of my tasks due to my current hell week, but I'll hopefully finish my current task by this weekend.

**Comment by yamayrie** [comment]
I didn't do much this week. Spent it thinking of what to write for a LTT

**Reply by JConcepcion01** [reply]
This week I I finished some tasks and worked mainly on one to grab the professor’s feedback and create the needed tasks to fix the next milestone

**Comment by IvanierB** [comment]
This week I completed a documentation task that improved sections 2.2.1 and 2.2.2 using the feedback given by the professor.

**Comment by dylanDeJesus21** [comment]
Was able to add descriptions to the project entities and eliminate section 1.4 from the Documentation
